tests/test_parity.py

```diff
@@ -90,3 +90,30 @@ def test_roberta_featurize(self):
             ),
             atol=1e-1
         )
+
+    def test_bert_featurize_fp16(self):
+        model = Classifier(base_model=BERT, float_16_predict=True)
+        np.testing.assert_allclose(
+            model.featurize(self.TEST_DATA)[0],
+            np.load(
+                os.path.join(
+                    DIRECTORY,
+                    'data/test-bert-activations.npy'
+                )
+            ),
+            atol=1e-1
```

Comment:
That tolerance seems quite high. Is that what we expect?

Reply:
It's the same tolerance we have for the main BERT/Roberta Models, but yes it is pretty high.